src/intel_batchbuffer.c

```diff
@@ -98,7 +97,7 @@ intel_batchbuffer_new(struct intel_driver_data *intel, int flag, int buffer_size
     assert(batch);
     batch->intel = intel;
     batch->flag = flag;
-    batch->run = drm_intel_bo_mrb_exec;
+    batch->run = drm_intel_gem_bo_fence_exec;
```

Comment:
Please update the minimum version requirement for libdrm in configure.ac and menson.build. The current version requirement of 2.4.52 doesn't support drm_intel_gem_bo_fence_exec